modules/ROOT/pages/configuring-properties.adoc

```diff
@@ -252,6 +252,22 @@ Configuration properties can be overwritten. The Mule runtime engine uses the fo
 
 So, for example, if a configuration property named `size` is defined as a system property, and there is also an application configuration property named `size`, the value for the application is the value of the property with the most precedence (in this case, the system property).
 
-Also, a property can derive its value from other properties that are higher in the hierarchy. Therefore, an application property can derive its value from environment, system, and deployment properties. A system property can derive its value from a deployment property, and so on. For example, if there is a system property named `env`, an application configuration property can have the value `file.$&#123;env&#125;.xml`.  On the other hand, an application property cannot depend on an application property's value unless it's defined in the same file. For example, a Secure Configuration property cannot depend on a Configuration property.
+Also, a property can derive its value from other properties that are higher in the hierarchy. Therefore, an application property can derive its value from environment, system, and deployment properties. A system property can derive its value from a deployment property, and so on. For example, if there is a system property named `env`, an application configuration property can have the value `file.$&#123;env&#125;.xml`. In addition, an application property cannot depend on an application property's value unless it's defined in the same file. For example, a Secure Configuration property cannot depend on a Configuration property.
 
 Because of this hierarchical structure, Mule components that load files cannot depend on an application or a global property. Mule runtime engine loads both, the component and the application property, at the same hierarchy. For example, if Mule loads an `import` component before loading the application property, the deployment fails because the property is not resolved by the time the `import` component needs to load.
+
+== Built-In Properties
```

Comment:
```suggestion
== Built-in Properties
```
Per SR, capitalize hyphenated words in titles exactly as you would if they were not hyphenated.